src/wealthsimple/wealthsimple.py

```diff
@@ -338,3 +338,163 @@ def get_forex(self) -> dict:
         """
         response = self.TradeAPI.makeRequest("GET", "forex")
         return response.json()
+
+    def get_security_id_from_stock_symbol(self, stock_symbol: str) -> str:
+        """Get Wealthsimple Security ID of a specific stock symbol (ticker)
+
+        Parameters
+        ----------
+        stock_symbol : str
+            Security symbol to search for
+
+        Returns
+        -------
+        str
+            A str containing Wealthsimple Security ID for the stock
+        """
+        response = self.TradeAPI.makeRequest("GET", f"securities?query={stock_symbol}")
+        response = response.json()
+        if response['total_count'] == 1:
+            # This check ensure we have an exact match for our search
+            return response["results"][0]['id']
+        else:
+            return ''
+
+    def place_order(self, order: dict) -> dict:
+
+        """Posts an order (market) to Wealthsimple API
+
+        Parameters
+        ----------
+        :param order:
+            The order dict containing the data to be submitted
+
+        Returns
+        -------
+        dict
+            A dict representing the  submitted order
+        """
+
+        response = self.TradeAPI.makeRequest("POST", "orders", order)
```

Comment:
How did you get this working with quantity? I've tried something very similar and was seeing 
{'error': 'quantity is invalid.'}